olm-catalog/serverless-operator/1.4.1/serverless-operator.v1.4.1.clusterserviceversion.yaml

```diff
@@ -445,5 +445,7 @@ spec:
       image: $IMAGE_KNATIVE_OPERATOR
     - name: knative-openshift-ingress
       image: $IMAGE_KNATIVE_OPENSHIFT_INGRESS
+    - name: knative-serving-operator
+      image: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-operator
```

Comment:
@jcrossley3 Should I replace the above k-s-o image with `$IMAGE_KNATIVE_SERVING_OPERATOR`?

Reply:
... with a corresponding change to `hack/catalog.sh`?

Reply:
Hmm, negative. That image comes "from the outside" and is not built by this repo.